catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1184,7 +1190,12 @@ public DeleteResponse delete(DeleteRequest deleteRequest)
                         .text(serializable.toString()));
             }
 
-            QueryImpl queryImpl = new QueryImpl(filterBuilder.anyOf(idFilters));
+            Filter anyTagFilter = filterBuilder.attribute(Metacard.TAGS)
```

Comment:
Since you use this twice, should this be extracted out into a `Supplier` function?
```java
Supplier<Filter> anyTagFilter = () -> filterBuilder.attribute(Metacard.TAGS).....;
```

and then here you can just  
```java
anyTagFilter.get()
```